infrastructure/critical/rds_id_minter.tf

```diff
@@ -72,7 +72,7 @@ module "identifiers_delta_rds_cluster" {
   password           = local.rds_password
 
   # This instance class provides us with 45 connections per instance.
-  instance_count = 3
+  instance_count = 1
```

Comment:
This was 2 rather than 1 before this change:
https://github.com/wellcomecollection/catalogue/commit/1bcdcb0f2f5e7c06e07631bc802c69c5900335e6#diff-9395daccebe509cdcf730a282c77aedd2bc345f6233d91c918f365b58d4e1c40L75

Reply:
:+1: